### DISCUSSION TITLE
Hola , cuando haga todas las Homeworks ... me anoto al Challenge ? y... en el Slack no pude encontrar el grupo de trabajo ..

### DISCUSSION THREAD

**Discussion by Joenstat** [atralice/Curso.Prep.Henry#192] [Q&A]
Hola , cuando haga todas las Homeworks ... me anoto al Challenge ? y... en el Slack no pude encontrar el grupo de trabajo ..

**Comment by Emmanuelvp88** [comment]
Tengo el mismo tema con Slack.

Att: Emmanuel villalva perez.

A quien coresponda.
